internal/util/grpcclient/client_test.go

```diff
@@ -49,4 +52,196 @@ func TestClientBase_connect(t *testing.T) {
 		assert.Error(t, err)
 		assert.True(t, errors.Is(err, ErrConnect))
 	})
+
+	t.Run("failed to get addr", func(t *testing.T) {
+		errMock := errors.New("mocked")
+		base := ClientBase{
+			getAddrFunc: func() (string, error) {
+				return "", errMock
+			},
+			DialTimeout: time.Millisecond,
+		}
+		err := base.connect(context.Background())
+		assert.Error(t, err)
+		assert.True(t, errors.Is(err, errMock))
```

Comment:
assert.Error may be redundant

Reply:
result err may be an error which is not `errMock`, we shall have assert to verify the case.